AnnouncementTableViewController.m

```diff
@@ -117,24 +123,42 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     
     Announcement *announcement = [announcements objectAtIndex:indexPath.row];
     
-    cell.category.text = [NSString stringWithFormat:@"  %@  ", [announcement getCategory]];
-    cell.category.backgroundColor = [self getColorForCategory:[announcement getCategory]];
-    [cell.category sizeToFit];
-    
     cell.title.text = [announcement getTitle];
     [cell.title sizeToFit];
     cell.title.adjustsFontSizeToFitWidth = YES;
 
-    cell.date.text = [announcement getNumberOfDaysInString];
+    cell.date.text = [announcement getDay];
+    [cell.date sizeToFit];
+    
+    cell.month.text = [announcement getMonth];
+    [cell.date sizeToFit];
+    
+    cell.year.text = [announcement getYear];
     [cell.date sizeToFit];
+
     
+    UIColor *color = [self getColorForCategory:[announcement getCategory]];
+    
+    UIImage *img = [UIImage imageNamed:@"WhiteBanner"];
+    cell.categoryImage.image = img;
+    cell.categoryImage.image = [img imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
+    cell.categoryImage.tintColor = color;
```

Comment:
Looks like we can just have `[self getColorForCategory:[announcement getCategory]]` here, instead of creating a new `UIColor` variable.